source/_integrations/overseerr.markdown

```diff
@@ -29,6 +32,44 @@ API key:
     type: string
 {% endconfiguration_basic %}
 
+## Supported functionality
+
+The Overseerr intergation provides a couple of entities to Home Assistant.
+Below is an overview of these entities.
+
+### Events
+
+Overseerr provides an event entity for updates around media.
+The possible events that this entity has are:
+ - `pending`
+ - `approved`
+ - `available`
+ - `failed`
+ - `declined`
+ - `auto_approved`
+
+Relevant data about the request are stored in the attributes.
+
+### Sensors
+
+The integration also provides statistics for the requests stored in Overseerr.
+There are sensors for:
+ - Total requests
+ - Movie requests
+ - TV requests
+ - Pending requests
+ - Declined requests
+ - Processing requests
+ - Available requests
+
+## Data updates
+
+When loading the integration, it will try to configure the webhook in Overseerr to give updates to Home Assistant.
+This makes the integration a push-based integration.
+
+When the integration receives an update about the requests, it will update the statistics to make sure they are up to date.
+Otherwise the integration will check for updates every 5 minutes.
```

Comment:
not sure I understand this. The integration gets updates from the devices, and in addition it polls the devices every 5 minutes?

Reply:
yep! its a local service so we can poll quite a lot

Reply:
oh, sorry. let me rephrase my question. What confuses me is the "Otherwise".. does it do both, in any case? or poll only if it hasn't heard anything from the device for the last 5 mins?

Reply:
Oh it does both simultaniously. I might change this in the future if the webhook seems to be stable enough. But yes, currently it does both at the same time